src/View.elm

```diff
@@ -1,31 +1,31 @@
 module View exposing (view)
 
-import Date
-import Date.Format exposing (format)
+import Time exposing (Time)
 import Html exposing (..)
 import Html.App
 import Html.Attributes exposing (id, for, attribute, class, type', value)
+import Utils exposing (timeAgo)
 import Model exposing (Model, Record, Msg(..))
 import Form
 
 
-formatLatsModified : Float -> String
-formatLatsModified timestamp =
-    format "%Y-%m-%d@%H:%M" (Date.fromTime timestamp)
+formatLastModified : Int -> Time -> String
+formatLastModified timestamp currentTime =
+    timeAgo (toFloat timestamp) currentTime
```

Comment:
I didn't get why this was required as `source-directories` includes `../src` and I thought the dependencies would be checked recursively, but I may be wrong. The thing is, tests pass, so they're definitely loaded appropriately. Now I don't quite know what to do with all this.